matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/pushrules/rest/PushRule.kt

```diff
@@ -104,7 +105,7 @@ data class PushRule(
      * Get the highlight status. As spec mentions assume false if no tweak present.
      */
     fun getHighlight(): Boolean {
-        return (getActions().firstOrNull { it is Action.Highlight } as? Action.Highlight)?.highlight ?: false
+        return (getActions().firstOrNull { it is Action.Highlight } as? Action.Highlight)?.highlight.orFalse()
```

Comment:
Why you did not take my whole suggestion?

Reply:
As sorry, read it too quickly